contracts/test/v0.8/dev/FunctionsBillingRegistry.test.ts

```diff
@@ -125,7 +125,7 @@ describe('OCR2DRRegistry', () => {
   // describe('General', () => {
   //   it('#typeAndVersion', async () => {
   //     expect(await registry.callStatic.typeAndVersion()).to.be.equal(
-  //       'OCR2DRRegistry 0.0.0',
+  //       'FunctionsRegistry 0.0.0',
```

Comment:
Should this be changed to `FunctionsBillingRegistry` so it matches the name of the contract?

Reply:
Probably. It's a disabled test currently because of contract size. But I'll change it just to be consistent.